guardian/typerighter
# Create rule manager client

## What does this change?
Adds a React/Typescript client to the rule manager app.

## How to test

In the root of the branch, run `sbt "ruleManager/run"` and navigate to https://typerighter.local.dev-gutools.co.uk/ where you should see the message "Hello world! This is the rule manager client!".

## How can we measure success?
Users will be (eventually) able to manage Typerighter rules through a website rather than having to manually edit a spreadsheet.
